ledfx/devices/twinkly_squares.py

```
@@ -0,0 +1,141 @@
+
+    def activate(self):
+        self.ctrl = xled.HighControlInterface(self._config["ip_address"])
+        try:
+            self.ctrl.turn_on()
+            self.ctrl.set_brightness(100)
+            self.ctrl.set_mode("rt")
+            info = self.ctrl.get_device_info()
+            _LOGGER.debug(
+                f"Twinkly Squares device {self.name} info: %s", info.data
+            )
+        except Exception as e:
```

Ruff (0.14.3)

Do not catch blind exception: `Exception`

(BLE001)